Eat_and_Greet/src/screens/OnboardingScreens/selectInterests.tsx

```diff
@@ -7,6 +7,20 @@ import CircularButton from '../../comps/CircularButton/CircularButton';
 
 const { width, height } = Dimensions.get('window');
 
+const interestArray = [ {title: "Park"},
```

Comment:
Love it, in the functional use we will need to pull the interests from some selection probably.